recipes/config.rb

```diff
@@ -27,6 +27,56 @@
   recursive true
 end
 
+directory '/etc/default' do
+  mode '0755'
+  owner 'root'
+  group 'root'
+  action :create
+  recursive true
+end
+
+# Setup /etc/default/impala
+if node['impala'].key?('config')
+  my_vars = { :options => node['impala']['config'] }
+
+  impala_log_dir =
+    if node['impala']['config'].key?('impala_log_dir')
```

Comment:
checking for lower case here.  default attributes are upper case (attributes are case-sensitive i thought?).  template does upcase anyway, so should probably just be consistent.